example/package-lock.json

```diff
@@ -56,7 +56,7 @@
     },
     "..": {
       "name": "@appcues/capacitor",
-      "version": "1.4.0",
+      "version": "2.0.0",
```

Comment:
I'm not sure why the capacitor version was on 1.4.0, when I ran npm run build it updated to 2.0 which I think is correct for this PR